### PR TITLE
[zh-cn]Update device-plugins.md

### PR DESCRIPTION
**Pull request by yanrongshi**
```yaml
content\zh-cn\docs\concepts\extend-kubernetes\compute-storage-net\device-plugins.md
```

### CODE REVIEW

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -579,11 +579,12 @@ Here are some examples of device plugin implementations:
 下面是一些设备插件实现的示例：
 
 * [AMD GPU 设备插件](https://github.com/RadeonOpenCompute/k8s-device-plugin)
-* [Intel 设备插件](https://github.com/intel/intel-device-plugins-for-kubernetes) 支持 Intel GPU、FPGA 和 QuickAssist 设备
+* [Intel 设备插件](https://github.com/intel/intel-device-plugins-for-kubernetes) 支持 Intel GPU、FPGA、QAT、VPU、SGX、DSA、DLB 和 IAA 设备
 * [KubeVirt 设备插件](https://github.com/kubevirt/kubernetes-device-plugins) 用于硬件辅助的虚拟化
 * [为 Container-Optimized OS 所提供的 NVIDIA GPU 设备插件](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
 * [RDMA 设备插件](https://github.com/hustcat/k8s-rdma-device-plugin)
 * [SocketCAN 设备插件](https://github.com/collabora/k8s-socketcan)
+* [SocketCAN 设备插件](https://github.com/collabora/k8s-socketcan)
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:587]: The translated text already exists. It's last contributor didn't update the commented text.

Comment (yanrongshi) [reply content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:587]: 好的，我当前应该将这个 pr 关闭对吗？前辈

Comment (Sea-n) [reply content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:587]: No, the previous line and commented lines still needs to be updated.

不用，只是有個小錯誤需要修正罷了，前一句中文、上方被註解的英文都還是需要更新的

Comment (windsonsea) [reply content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:587]: Line 586 and 587 are duplicated. Delete one please.

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 2]
```diff
@@ -579,7 +579,7 @@ Here are some examples of device plugin implementations:
 下面是一些设备插件实现的示例：
 
 * [AMD GPU 设备插件](https://github.com/RadeonOpenCompute/k8s-device-plugin)
-* [Intel 设备插件](https://github.com/intel/intel-device-plugins-for-kubernetes) 支持 Intel GPU、FPGA 和 QuickAssist 设备
+* [Intel 设备插件](https://github.com/intel/intel-device-plugins-for-kubernetes) 支持 Intel GPU、FPGA、QAT、VPU、SGX、DSA、DLB 和 IAA 设备
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:582]:
```suggestion
* [Intel 设备插件](https://github.com/intel/intel-device-plugins-for-kubernetes)支持 Intel GPU、FPGA、QAT、VPU、SGX、DSA、DLB 和 IAA 设备
```

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 3]
```diff
@@ -579,11 +579,12 @@ Here are some examples of device plugin implementations:
...
 
 * [AMD GPU 设备插件](https://github.com/RadeonOpenCompute/k8s-device-plugin)
-* [Intel 设备插件](https://github.com/intel/intel-device-plugins-for-kubernetes) 支持 Intel GPU、FPGA 和 QuickAssist 设备
+* [Intel 设备插件](https://github.com/intel/intel-device-plugins-for-kubernetes)支持 Intel GPU、FPGA、QAT、VPU、SGX、DSA、DLB 和 IAA 设备
 * [KubeVirt 设备插件](https://github.com/kubevirt/kubernetes-device-plugins) 用于硬件辅助的虚拟化
 * [为 Container-Optimized OS 所提供的 NVIDIA GPU 设备插件](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
 * [RDMA 设备插件](https://github.com/hustcat/k8s-rdma-device-plugin)
 * [SocketCAN 设备插件](https://github.com/collabora/k8s-socketcan)
+* [SocketCAN 设备插件](https://github.com/collabora/k8s-socketcan)
 * [Solarflare 设备插件](https://github.com/vikaschoudhary16/sfc-device-plugin)
```
[the remark this answers is not in the document]
Comment (yanrongshi) [reply content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: > 还是没有修好。

抱歉，我在推送的时候发生了错误

**Review by Sea-n — COMMENTED** [review]
Thanks. 😄
/approve

**Pull request merged**